src/applications/vaos/api/index.js

```diff
@@ -293,7 +293,7 @@ export function getRequestLimits(facilityId, typeOfCareId) {
 export function getAvailableClinics(facilityId, typeOfCareId, systemId) {
   let promise;
   if (USE_MOCK_DATA) {
-    if (facilityId === '983A6') {
+    if (facilityId === '983') {
```

Comment:
Is this related to slots? Or just a separate fix you caught?

Reply:
The DS flow wasn't working for me locally, this fixed it.